src/pages/en/guides/cms/wordpress.md

```diff
@@ -2,12 +2,187 @@
+  ))
+}
+```
+
+The WordPress REST API includes [global parameters](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/) such as `_fields` and `embed`. 
+
+A large quantity of data is available to you via this API, so you may wish to only fetch certain fields. You can restrict your response by adding the `[_fields`](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_fields) paramater to the API URL, for example: `[YOUR-SITE]/wp/v2/posts?_fields=author,id,excerpt,title,link` 
+
+The API can also return content related to your post, such as a link to the parent post, or to comments on the post. You can add the [`_embed`](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_embed) parameter to the API URL (e.g. `[YOUR-SITE]/wp/v2/posts?_embed`) to indicate to the server that the response should include these embedded resources.
+
+## Building a blog with WordPress and Astro
+
+This example fetches data from the public WordPress API of [https://norian.studio/dinosuars/](https://norian.studio/dinosaurs/). This WordPress site stores information about individual dinosaurs under the `dinos` route, just as a blog would store individual blog posts under the `posts` route.
+
+This example shows how to reproduce this site structure in Astro: an index page that lists dinosaurs with links to dynamically-generated individual dinosaur pages.
+
+:::note
+To use [Custom Post Types (CPT)](https://learn.wordpress.org/lesson-plan/custom-post-types/) in your WordPress API (not just `post` and `page`), you will have to [configure them in your WordPress dashboard](https://stackoverflow.com/questions/48536646/how-can-i-get-data-from-custom-post-type-using-wp-rest-api) or [add REST API Support For Custom Content Types](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-rest-api-support-for-custom-content-types/) in WordPress.
+
+This example fetches data from a WordPress site whose content types have already been configured and exposed to the REST API.
+:::
+
+
+### Displaying a list of WordPress posts
+
+The page `src/pages/index.astro` lists each dinosaur, with a description and link to its own page.
+
+```ini title="Project Structure" {3}
+├── src/
+├── pages/
+│    └── index.astro
+│    └── dinos/
+│         └── [slug].astro
+├── astro.config.mjs
+└── package.json
+```
+
+Fetching via the API returns an object that includes the properties:
+- `title.rendered` - Contains the HTML rendering of the title of the post.
+- `content.rendered` - Contains the HTML rendering of the content of the post.
+- `slug` - Contains the slug of the post. (This provides the link to the dynamically-generated individual dinosaur pages.)
+
+```astro title="/src/pages/index.astro"
+---
+import Layout from "../layouts/Layout.astro"
+
+let res = await fetch("https://norian.studio/wp-json/wp/v2/dinos")
+let posts = await res.json();
+---
+
+<html lang="en">
```

Comment:
```suggestion
```
Removing the `<html>` tag here, since we normally take care of that in a typical layout.